test/y2partitioner/widgets/reprobe.rb

```diff
@@ -0,0 +1,53 @@
+# encoding: utf-8
```

Comment:
NP: We could move this file to `test/support`, due to we have put there other examples files.